src/qutip_tensorflow/core/data/dense_tensor.py

```diff
@@ -0,0 +1,85 @@
+import qutip as qt
```

Comment:
again, really not sure I see a reason to abbreviate this.  People read code more than they write it.